samples/lightweight-component/calc_pipeline.py

```diff
@@ -0,0 +1,81 @@
+import kfp
+import kfp.components as comp
+
+#Define a Python function
+def add(a: float, b: float) -> float:
+   '''Calculates sum of two arguments'''
```

Comment:
nit pick: [PEP-257](https://www.python.org/dev/peps/pep-0257/) recommends triple-double quotes surrounding docstrings

Reply:
Thanks @ckadner, many of the kfp tutorial samples are using single quote so I was trying to be consistent. But i will update it to double quotes since Python has better coding style.

Reply:
Done